app/lib/publisher/backend.dart

```diff
@@ -93,6 +96,40 @@ class PublisherBackend {
     return _asPublisherInfo(p);
   }
 
+  /// Invites a user to become a publisher admin.
+  Future<account_api.InviteStatus> invitePublisherMember(
+      String publisherId, api.InviteMemberRequest invite) async {
+    return await _withPublisherAdmin(publisherId, (p) async {
+      InvalidInputException.check(
```

Comment:
Add some sanity limits..
```dart
InvalidInputException.checkNotNull(invite.email, 'email');
InvalidInputException.checkStringLength(invite.email, 'email', maximum: 4096);
InvalidInputException.checkNotNull(invite.reason, 'reason');
InvalidInputException.checkStringLength(invite.reason, 'reason', maximum: 4096);
```

Reply:
I'm not using the reason String for anything. Maybe we can remove it? (Why would an admin state a reason to invite somebody else?)

Reply:
fair enough, let's remove it...

I was thinking we would surface it in an email to all the other members.